backend/qmlscene/qmlscene.go

```diff
@@ -6,11 +6,7 @@
 //
 // In simple cases, an application can execute with:
 //
-//	type Root struct {
-//		qbackend.QObject
-//	}
-//
-//	qmlscene.Connection.RootObject = &Root{}
+//	qmlscene.Connection.RegisterType(...)
```

Comment:
Idle note.. I want to stress somewhere that it isn’t necessary to register every type that gets sent to QML — especially because there’s a cap on the number of these.

Can do that when we’re documenting singletons